arrocke/gloss-translation
# refactor: update glosses and notes using a phrase id

## What has changed

* Create phrases for each word in a verse when loading the translation view
* Update the gloss and notes associated with a phrase ID rather than a word ID



## Connected Issues

next step for #361 



## QA Steps



1. Try updating a gloss, and both kinds of notes.
2. Try approval action
3. Check that gloss history entry records are created for each action

## Post-Deployment
